### PR TITLE
hide promo banner

### PR DESCRIPTION
**Pull request by edunham**
This reverts commit 1211b97a71e259bd1601a363769b8503884f5ffa.

This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL approved by Dev Advocacy
- [ ] The content has been run through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist
- [ ] Images are compressed appropriately
- [ ] Social image previews well on Twitter and LinkedIn
- [ ] Tech review request (developer advocate or domain expert)
- [ ] Review request for editorial/grammar/clarity (developer advocate or Colton)

### CODE REVIEW

**Review by isemona — APPROVED** [review]

**Review by alisaduncan — CHANGES_REQUESTED** [review]
Instead of reverting the PR, can we just hide the banner please?

**Review by alisaduncan — COMMENTED** [review]
Changing to comment so it doesn't block merge, but respectfully requesting changing the flag instead of reverting the PR.

File: _source/_includes/promo_banner.html [hunk 1]
```diff
@@ -8,11 +8,11 @@
 {% endfor %}
 
 {% if show_promo_banner %}
-  <a href="/blog/2024/09/26/oktane-saas-developers" class="DocsPromoBanner">
+  <a href="https://regionalevents.okta.com/enterprisereadyworkshops" class="DocsPromoBanner">
```
Comment (edunham) [inline _source/_includes/promo_banner.html]:
```suggestion
  <a href="/blog/2024/09/26/oktane-saas-developers" class="DocsPromoBanner">
```

File: _source/_includes/promo_banner.html [hunk 2]
```diff
@@ -8,11 +8,11 @@
...
-      <span class="promo-body">See you at Oktane in Las Vegas on October 15-17, 2024. Read more about the activities planned with you mind here.</span>
+      <span class="promo-body">Check out the free virtual workshops on how to take your SaaS app to the next level in the enterprise-ready identity journey!</span>
```
Comment (edunham) [inline _source/_includes/promo_banner.html]:
```suggestion
      <span class="promo-body">See you at Oktane in Las Vegas on October 15-17, 2024. Read more about the activities planned with you mind here.</span>
```

**Comment by edunham** [comment]
swapped it over to just hide the banner. I'd assumed we might want to resume promo on the workshops soonish, but I have no objections to leaving the rest in the oktane promo state for now.

**Review by alisaduncan — APPROVED** [review]

**Comment by isemona** [comment]
confirmed with @edunham good to close

**Pull request merged**